server.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+let connect = require("connect");
```

Comment:
Single quotes are generally preferred in javascript. I am not sure if your Jshint configurations were recommended. If so then it's okay as long as it is consistent.